Add release blog post for 0.13.1

harding opened the pull request:
This adds a release blog post for 0.13.1.

**TODO**

- [ ] This assumes the release date is Monday 31 Oct 2016.  If it as any other date, the file needs to be moved and the YAML header needs to edited.
- [ ] At the very top of the document, a link needs to be added to @laanwj's PGP-signed release announcement (as we did with the previous release).  The tests should fail until this is added, but I see they succeeded, which I guess means I'll have to fix that...
- [ ] The SHA256 hashes of the release files should be added to the bottom of this document (the "Hashes for verification" section) as was done for the previous release.
- [x] https://github.com/bitcoin-core/bitcoincore.org/pull/240 needs to be merged before this, or this document needs to be edited to stop linking to it.  (The tests should also be failing because that's missing in this branch...)

---

A preview for the blog post as of fd373869eab2e473c7953efe3098d2d604a2f9c3 may be found at: http://imgur.com/a/gKi9W

The post entry on the main page looks like this:

![2016-10-26-21_01_20_253721417](https://cloud.githubusercontent.com/assets/61096/19750483/6613a372-9bbf-11e6-8b5a-36d88d3e100d.png)

Please note that if you attempt to generate a preview of this document, you will need to add `--future` to Jekyll's parameters because the post is dated for 2016-10-31.

Several people provided suggestions on this in draft form, for which I am very grateful. Any errors that appear in this version are entirely my fault.  For anyone who read the final draft version, I've made a few small changes:

1. I further shortened some of the descriptions in the new features summary section at the top.  The only significant phrasing change is in point 7 (More efficient almost-full-node security).  I added a few paragraph breaks in the corresponding detail subsections but no words were changed there.

2. To the end of the document, I added a new Conclusion section (and also the template Hashes For Verification section) to match the format of the 0.13.0 release blog.

3. I changed the opening to the null dummy section to make it flow a little better.

As requested in some of the draft feedback, I added the ability to link directly to each of the items on the timeline.  This is a little bit ugly, so I'm open to suggestions for making it look nicer.  Screenshot below (the `#` are both the link and the anchor):

![2016-10-26-21_07_22_112758683](https://cloud.githubusercontent.com/assets/61096/19750577/393364d6-9bc0-11e6-9240-ea88e9dedf75.png)

btcdrak commented on _posts/en/posts/2016-10-31-release-0.13.1.md:
```diff
@@ -0,0 +1,171 @@
+
+excerpt: >
+  The first release of Bitcoin Core that supports activation of
+  segregated witness is now available.
```
This should be on one line for translators

btcdrak commented on _posts/en/posts/2016-10-31-release-0.13.1.md:
```diff
@@ -0,0 +1,171 @@
+    FIXME
+
+
+[segwit upgrade guide]: /en/segwit-upgrade-guide/
```
This link has been changed to /en/2016/10/27/segwit-upgrade-guide/

kanzure replied:
`/en/segwit-upgrade-guide/` is currently incorrect here.

btcdrak commented:
Can you also add a copy of the release notes?

Michagogo commented:
@btcdrak AIUI there's supposed to be a link to them at the top, once @laanwj pgp-signs and posts them.

Michagogo commented:
Unless the release announcement is something else?

Michagogo commented:
No, looks like it is indeed the ML post with the release notes.

btcdrak commented:
@Michagogo No, the release notes are not signed, you're thinking of the release announcement which has the signed hashes. The release notes are separate, check "releases" on the website.

Michagogo commented:
Ah, I see.

harding commented:
Fixed feedback from @btcdrak and @kanzure (thanks!) and rebased on top of merge that includes the segwit upgrade guide (and checked the preview and cross-link to ensure they worked).

harding commented:
Also just added release notes.  This is a copy of https://raw.githubusercontent.com/bitcoin/bitcoin/2e2388a5cbb9a6e101b36e4501698fec538a5738/doc/release-notes/release-notes-0.13.1.md with the following global substitution to fix a formatting difference between GitHub-flavor Markdown and the Markdown used on this site: `s/^- #/- \\#/`  (e.g. the list items that start with a `#` are rendered as subheadings on BitcoinCore.org; this prefaces them with a backslash to have them render normally as `#`)

btcdrak commented:
OK released, pulling the trigger now.
